Cargo.toml

```diff
@@ -45,12 +45,15 @@ dirs = "5.0.1"
 indoc = { version = "2.0.4", optional = true }
 log = { version = "0.4.8", default-features = false }
 miette = { version = "5.7.0", features = ["fancy"] }
+notify = { version = "8.1.0", optional = true }
+notify-debouncer-mini = { version = "0.4", optional = true }
 once_cell = "1"
 parking_lot = "0.12"
 radix_trie = "0.2"
 rustc-hash = "1.1.0"
 simplelog = "0.12.0"
 serde_json = { version = "1", features = ["std"], default-features = false, optional = true }
+tempfile = { version = "3.8", optional = true }
```

Comment:
This should be in a `[dev-dependencies]` section because it is only used for tests. Though the tests are all pointless anyway, so just remove this.